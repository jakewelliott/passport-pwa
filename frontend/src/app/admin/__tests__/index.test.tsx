import { describe, expect, it } from 'vitest';
import { AdminPage } from '../index';

import { setupTestEnv } from '@/lib/testing/test-wrapper';

const { render } = setupTestEnv();

describe('AdminPage', () => {
    it('matches snapshot', () => {
        const { container } = render(<AdminPage />);
        expect(container).toMatchSnapshot();
    });
});
