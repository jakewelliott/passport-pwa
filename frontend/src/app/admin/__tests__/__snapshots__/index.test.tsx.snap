// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AdminPage > matches snapshot 1`] = `
<div>
  <div>
    <div
      class="flex flex-col gap-3.5"
      data-testid="more-list"
    >
      <a
        class="text-supporting-inactiveblue no-underline"
        data-discover="true"
        href="/admin/edit-parks"
      >
        <div
          class="flex items-center bg-supporting-lightblue p-3 shadow-md"
          style="border-radius: 15px;"
        >
          <h2
            class="mx-4 my-6"
          >
            Edit Parks
          </h2>
        </div>
      </a>
      <a
        class="text-supporting-inactiveblue no-underline"
        data-discover="true"
        href="/admin/edit-trails"
      >
        <div
          class="flex items-center bg-supporting-lightblue p-3 shadow-md"
          style="border-radius: 15px;"
        >
          <h2
            class="mx-4 my-6"
          >
            Edit Trails
          </h2>
        </div>
      </a>
      <a
        class="text-supporting-inactiveblue no-underline"
        data-discover="true"
        href="/admin/edit-bucket-list"
      >
        <div
          class="flex items-center bg-supporting-lightblue p-3 shadow-md"
          style="border-radius: 15px;"
        >
          <h2
            class="mx-4 my-6"
          >
            Edit Bucket List
          </h2>
        </div>
      </a>
      <a
        class="text-supporting-inactiveblue no-underline"
        data-discover="true"
        href="/admin/edit-users"
      >
        <div
          class="flex items-center bg-supporting-lightblue p-3 shadow-md"
          style="border-radius: 15px;"
        >
          <h2
            class="mx-4 my-6"
          >
            Edit Users
          </h2>
        </div>
      </a>
      <label
        class="cursor-pointer text-supporting-inactiveblue no-underline"
      >
        <div
          class="flex items-center bg-supporting-lightblue p-3 shadow-md"
          style="border-radius: 15px;"
        >
          <h2
            class="mx-4 my-6"
          >
            Upload GeoJson Data
          </h2>
        </div>
        <input
          class="hidden"
          type="file"
        />
      </label>
    </div>
  </div>
</div>
`;
